src/Controller/OrganizationController.php

```diff
@@ -69,14 +69,23 @@ public function overview(Organization $organization): Response
      */
     public function packages(Organization $organization, Request $request): Response
     {
-        $count = $this->packageQuery->count($organization->id());
+        $filter = Filter::fromRequest($request);
+
+        $count = $this->packageQuery->count($organization->id(), $filter);
         $user = parent::getUser();
-        if ($count === 0 && $user instanceof User && $organization->isOwner($user->id())) {
+
+        // If the filtered version returned 0 results, we need to count again, but without any filters
+        if ($count === 0 &&
+            $this->packageQuery->count($organization->id(), (new Filter())) === 0 &&
+            $user instanceof User &&
+            $organization->isOwner($user->id())
```

Comment:
maybe we can find a name for this arguments and extract private method, for example `isNoPacakges()` or `suggestAddingPackage()`